packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -195,7 +195,17 @@ class _ProductListPageState extends State<ProductListPage> {
                   );
                   if (dismissible) {
                     return Dismissible(
-                      background: Container(color: colorScheme.background),
+                      //Delete Animation Added
+                      direction: DismissDirection.endToStart,
+                      background: Container(
+                        alignment: Alignment.centerRight,
+                        padding: const EdgeInsets.only(right: 30),
+                        color: Colors.red,
```

Comment:
We have our own color scheme in design_constants.dart it would be good if you could switch to that color

Reply:
I have now changed Colors.red to RED_COLOR from design_constants.dart